emission/pipeline/intake_stage.py

```diff
@@ -198,7 +198,13 @@ def run_intake_pipeline_for_user(uuid, skip_if_no_new_data):
         esds.store_pipeline_time(uuid, ecwp.PipelineStages.CREATE_COMPOSITE_OBJECTS.name,
                                  time.time(), crt.elapsed)
 
-        _get_and_store_range(uuid, "analysis/composite_trip")
+        with ect.Timer() as gsr:
+            logging.info("*" * 10 + "UUID %s: generating store and range " % uuid + "*" * 10)
+            print(str(arrow.now()) + "*" * 10 + "UUID %s: generating store and range " % uuid + "*" * 10)
+            _get_and_store_range(uuid, "analysis/composite_trip")
+
+        esds.store_pipeline_time(uuid, 'GENERATE_STORE_AND_RANGE',
+                                 time.time(), gsr.elapsed)
```

Comment:
@TeachMeTW this is fine. As I suggested, can you please pull this out to a separate PR so that I can accept and merge it immediately? This will unblock us for collecting baseline data for this function before refactoring it next week.

From https://github.com/e-mission/e-mission-server/pull/990#issuecomment-2452811296

> You can even create two PRs so I can merge the easy one while reviewing the more complex one.

or https://github.com/e-mission/e-mission-server/pull/990#issuecomment-2453223948

> In particular, "easy: one for adding instrumentation to the _get_and_store_range?" is the main change that we want to get into production soon.

Reply:
@shankari see #991 991